src/set.js

```diff
@@ -0,0 +1,71 @@
+export default class Set {
+  constructor() {
+    this.data = [];
+    this.length = 0;
+  }
+
+  add( stuff ) {
+      if( this.data.length > 0 ){
+        for( var i = 0; i < this.data.length; i++ ){
+          if( this.data[i] == stuff ){
+            return "ERROR, " + stuff + " already exists"
+          }else{
+          return this.data.push(stuff)
+          }
+        }
+      }else{
+        return this.data.push(stuff)
+      }
+  }
+
+  isEmpty() {
+    if(this.data.length > 0) {
+    return false
+    }else{
+    return true
+    }
+  }
+
+  contains(stuff) {
+    for(var i = 0; i < this.data.length; i++ ) {
+      if(this.data[i] == stuff) {
```

Comment:
this can just be `return this.data[i] === stuff`
generally default to `===` ! 
double equal can yield unexpected results and introduce hard bug.s